src/datapump/globals.py

```diff
@@ -58,5 +58,11 @@ class Globals(EnvSettings):
     max_versions: int = Field(4, env="MAX_VERSIONS")
     datapump_table_name: Optional[str] = Field(env="DATAPUMP_TABLE_NAME")
 
+    gcs_key_secret_arn: Optional[str] = Field(None, env="GCS_KEY_SECRET_ARN")
+
+    google_application_credentials = Field(
+        "/root/.gcs/private_key.json", env="GOOGLE_APPLICATION_CREDENTIALS"
```

Comment:
Have you tested this out on an actual lambda function on AWS yet? I think you can only write to the /tmp folder from the lambda.

Reply:
I did not know that, thanks! Will change it to write under /tmp before committing.